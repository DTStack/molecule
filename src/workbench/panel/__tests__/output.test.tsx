import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';

import Output from '../output';

describe('Test Output Component', () => {
    test('Match the Output snapshot', () => {
        const component = renderer.create(<Output id="output" />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
