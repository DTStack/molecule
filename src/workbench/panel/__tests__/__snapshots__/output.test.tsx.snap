// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Output Component Match the Output snapshot 1`] = `
<div
  className="mo-output"
>
  <div
    className="mo-monaco-editor"
    style={
      Object {
        "height": "100%",
        "minHeight": "400px",
        "position": "relative",
        "width": "100%",
      }
    }
  />
</div>
`;
